src/main/resources/assets/tconstruct/book/puny_smelting/en_us/upgrades/ranged/sinistral.json

```diff
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:sinistral",
+  "text": [
+    {
+      "text": "Double the bows double the trouble."
+    }
+  ],
+  "effects": [
+    "Crossbows in main hand will fire with attack button, and will not reload when sneaking",
```

Comment:
If you have space, be worth adding a sentence explaining the why, as the details do not make it obvious.

Something like "Allows dual wielding crossbows, or using a crossbow with tools like shields"